content/blog/posts/o3de-introducing-design-system.md

```diff
@@ -0,0 +1,95 @@
+* **Accessibility:** Accessibility options, such as color contrasts, fonts and icon legibility, interactions and scalability etc. are built into our the tool from the first step.
+* **Familiarity:** Align to our standards of the visual interfaces, tools, branding, and UI elements. No need to reinvent the wheel and confuse users.
+* **Modularity:** The key to each element of O3DE, including the interface design and component development.
+
+These tenets were formed to be straightforward and easy to remember. We want to ensure that the BJDS tenets are flexible for the community to embrace, and to employ through their contributions to the engine.
+
+### What's included in the BlueJay Design System
+
+**The BlueJay Design System** is made up of a set of standardized components and layouts to help accelerate the creation of new workflows for the editor. It also has [supporting documentation](https://o3de.org/docs/tools-ui/) on the O3DE Learn website, which details the following:
+
+#### UI Components
+
+UI components are the various individual widgets that make up the editor experience. The BJDS documentation outlines details like visual markup properties, rules around how to use them and guides when to use them in which scenario. Currently, with [20 UI components](https://o3de.org/docs/tools-ui/component-library/) detailed (and more coming), these include code samples and API references to integrate into your editor extension.
+
+If you are looking for information on key building blocks that O3DE uses, [UI Components](https://o3de.org/docs/tools-ui/component-library/) is the right place to read. It's the systematic reuse of these components that make up the visual and functional consistency across O3DE.
+
+![Preview @mdash; for the O3DE UI component guidelines](/images/blog/intro-to-bjds/ui-components-preview.jpg)
+
+#### UX Patterns
+
+UX Patterns are a series of best practice solutions that a designer or developer should follow when creating new workflows and components in the editor. Unlike UI Components, where it's mostly about the visual aesthetics and how it mechanically operates, UX Patterns is more about how a user interacts and behaves for a task with a goal in mind.
+
+For e.g., the anatomy of a component card and it's different types can be found on the [UI Component > Card page](https://o3de.org/docs/tools-ui/component-library/uidev-card-component/), while the what contents go in the card, how a user interacts with it and how O3DE interacts with the card itself can be found on the [UX Patterns > Component Card UX Patterns](https://o3de.org/docs/tools-ui/ux-patterns/component-card/overview/) page.
+
+![Preview image for the O3DE component card guidelines](/images/blog/intro-to-bjds/componentcard-ux-preview.jpg)
+
+Currently we have two UX Pattern pages on O3DE Learn: Component Card Properties and Writing UI Message guidelines. We plan to roll out more pages in the future coming months, so stay tuned.
```

Comment:
Along with the image above, the UX Pattern pages your referring to seems to point at "Component Card > Overview" and "Component Card > Writing Guidelines". But I'm guessing in the image, you're referring to the "Component Card" and the "Error Handling" in the left-hand TOC. 

I think you can clarify which pages you mean by adjusting the titles you list here. 

```suggestion
Currently we have two UX Pattern pages on O3DE Learn: Component Card and UI Messages for Error Handling. We plan to roll out more pages in the coming months, so stay tuned.
```